stable_plugins/classical_ml/data_preparation/sym_max_mean.py

```diff
@@ -374,10 +404,14 @@ def calculation_task(self, db_id: int) -> str:
 
     zip_file.close()
 
+    concat_filenames = retrieve_filename_from_url(entities_url)
+    concat_filenames += retrieve_filename_from_url(element_similarities_url)
+    info_str = f"_from_{get_readable_hash(concat_filenames)}"
```

Comment:
Similarly to the function ``retrieve_filename_from_url`` I am using the function ``get_readable_hash`` often. Should this function also become an API function in the plugin runner?

Reply:
We could do that. The reason why I did not propose that in the first place was that it would introduce another dependency. But you are right in that it is used often and can be used by other plugins as well. (You can decide if you want to do the work.)

Reply:
I am going to leave it for now